Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:latest
+MAINTAINER nelluri@deis.com
+RUN apt-get update && apt-get -y install make python lsb-release unzip wget vim git curl jq
+RUN wget https://storage.googleapis.com/kubernetes-release/release/v1.0.3/bin/linux/amd64/kubectl
+RUN chmod +x kubectl
+RUN mv kubectl /usr/local/bin/
+WORKDIR /home
+RUN wget --no-check-certificate https://dl.bintray.com/deis/helm/helm-0.5.0+1689ee4-linux-amd64.zip
+RUN unzip helm-0.5.0+1689ee4-linux-amd64.zip
+RUN mv helm /usr/local/bin/
+ENV CLOUD_SDK_REPO cloud-sdk-precise
+RUN echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | tee /etc/apt/sources.list.d/google-cloud-sdk.list
+RUN curl -Lk https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+RUN apt-get update && apt-get install google-cloud-sdk
```

Comment:
Once again - please put all the apt-get related logic into one block at the top of the file:

- add apt keys
- add apt sources
- run apt-get update once
- install all apt packages